sdk/core/azure-core/src/main/java/com/azure/core/util/configuration/Configuration.java

```diff
@@ -24,13 +24,14 @@
 
     private final ClientLogger logger = new ClientLogger(Configuration.class);
 
-    private ConcurrentMap<String, String> configurations = new ConcurrentHashMap<>();
+    private final ConcurrentMap<String, String> configurations;
```

Comment:
The instantiation could have been kept on the same line.
```java
private final ConcurrentMap<String, String> configurations = new ConcurrentHashMap<>();
```

Reply:
there's a constructor that sets the value for this field
```
    private Configuration(ConcurrentMap<String, String> configurations) {
        this.configurations = new ConcurrentHashMap<>(configurations);
    }
```

and another constructor that don't
```
  public Configuration() {
      
    }
```

If we want field to be final, we need to let constructor handle assignation. 
Or do you have something else on mind that I might not considering?

Reply:
Ahh gotcha. That makese sense. I didn't see the other constructor.